### PR TITLE
Corrections for updates to chapters 4 & 5

### PR DESCRIPTION
**Pull request by jonmcalder**

### CODE REVIEW

File: 04-Working-with-Unix.Rmd [hunk 1]
```diff
@@ -896,7 +896,7 @@ you can use the `find` command. This command has a specific structure where
 the first argument is the directory where you want to begin the search, and all
 directories contained within that directory will also be searched. The first
 argument is then followed by a flag that describes the method you want to use to
-search, in this case we'll only be searching for a file by its name, so we'll
+search. In this case we'll only be searching for a file by it's name, so we'll
```
Comment (seankross) [inline 04-Working-with-Unix.Rmd]: @jonmcalder isn't "its name" correct?

**Comment by seankross** [comment]
@jonmcalder thank you for doing this!

File: 04-Working-with-Unix.Rmd (hunk 1 above)
Comment (jonmcalder) [inline 04-Working-with-Unix.Rmd]: Whoops!  Yes "its name" is correct.

Comment (seankross) [inline 04-Working-with-Unix.Rmd]: Okay just making sure, clearly spelling and grammar are not my strongest suits :)

**Pull request merged**